### ISSUE TITLE
Build Broken for .Net

### ISSUE THREAD

**Issue by fly-man-** [WhiteCoreSim/WhiteCore-Dev#314]
"C:\projects\whitecore-dev\WhiteCore.sln" (default target) (1) ->
"C:\projects\whitecore-dev\WhiteCore\BotManager\WhiteCore.BotManager.csproj" (default target) (17) ->
(CoreCompile target) -> 
  Bot_API.cs(233,50): error CS0103: The name '_' does not exist in the current context [C:\projects\whitecore-dev\WhiteCore\BotManager\WhiteCore.BotManager.csproj]

**Comment by emperorstarfinder** [comment]
Yes but I think it is AppVeyor because it is an _ this time in greythane's latest commit.  When I pushed a commit to my team's project last night I found AppVeyor hated a , which was necessary and did not break build locally.

**Comment by greythane** [comment]
Just rebuilt on Mac and Win 10 from scratch with no errors. 
I will recheck to see what is being referred to though in the morning. (11pm)
